lib/Dialect/TritonNvidiaGPU/Transforms/TMALowering.cpp

```diff
@@ -14,18 +14,15 @@
 #include "triton/Dialect/TritonNvidiaGPU/Transforms/TMAUtilities.h"
 #include "llvm/Support/ErrorHandling.h"
 
-#include <memory>
+namespace mlir {
+namespace triton {
+namespace nvidia_gpu {
 
-#define GEN_PASS_CLASSES
+#define GEN_PASS_DEF_TRITONNVIDIAGPUTMALOWERINGPASS
 #include "triton/Dialect/TritonNvidiaGPU/Transforms/Passes.h.inc"
 
 namespace {
 
-using namespace mlir;
```

Comment:
re the changes with using namespace related? There is not much consistency so I don't understand the point of changing it.

Reply:
Yes, there is a slight connection. Since I now use namespace `mlir::triton::nvidia_gpu` everywhere, `using namespace` is usually no longer needed (except for `triton::gpu` namespace). This file really has an inconsistent approach. It would be better to use it like `namespace ttg = mlir::triton::gpu;`. WDYT?